src/request.rs

```diff
@@ -308,6 +162,22 @@ pub enum Request {
     },
     /// Get current wallet balance
     GetBalance(XorName),
+    /// Create a new coin balance
+    CreateCoinBalance {
+        source: XorName,
```

Comment:
Could you please remove the `source` field in this request?
It was suggested in the review for https://github.com/maidsafe/safe-nd/pull/43.
So the `source` will be derived from the requester's public key.

Reply:
These changes are already made in the same PR#43 . We will rebase this once #43 gets merged.